lib/js_of_ocaml/dom_html.ml

```diff
@@ -2165,6 +2165,37 @@ class type history =
 
 class type undoManager = object end
 
+class type optionPromise =
```

Comment:
I think this should be named differently

Reply:
So what name should I use instead? (I based this name from the object name "optionEvent")

Reply:
`optionEvent` ? where ?
If theses options are about `share`, should it be `shareOption` maybe ?